travis/publish.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+echo "********************************"
+echo "***** Publishing Artifacts *****"
+echo "********************************"
+echo "Branch: ${TRAVIS_BRANCH}"
+echo "Tag: ${TRAVIS_TAG}"
+echo "JDK Version: ${TRAVIS_JDK_VERSION}"
+
+#Weird comparison comparing branch to tag because when you 'git push --tags'
+#the branch somehow becomes the tag value
+#github issue: https://github.com/travis-ci/travis-ci/issues/1675
+test "${TRAVIS_BRANCH}" = "${TRAVIS_TAG}" &&
+test "${TRAVIS_TAG}" != "" &&
+test "${TRAVIS_JDK_VERSION}" = 'oraclejdk7' &&
+sh "$TRAVIS_BUILD_DIR/.travis-ci-bintray-credentials.sh" &&
+sbt 'set version := version.value + "." + System.getenv("TRAVIS_BUILD_NUMBER")' publish
```

Comment:
I think this is where you want to parse the tag to get the version number, or use a plugin that does that for you.